modules/account-lib/src/coin/algo/errors.ts

```diff
@@ -16,9 +16,31 @@ export class AddressValidationError extends BuildTransactionError {
     this.name = AddressValidationError.name;
   }
 }
+
 export class KeyDilutionError extends BuildTransactionError {
   constructor(incorrectValue: number) {
     super(`The keyDilution value '${incorrectValue}' must be less than or equal to the square root of the voteKey validity range.`);
     this.name = KeyDilutionError.name;
   }
-}
\ No newline at end of file
+}
+
+export class PublicKeyValidationError extends InvalidKey {
```

Comment:
we don't need these. see: https://github.com/BitGo/BitGoJS/blob/master/modules/account-lib/src/coin/baseCoin/errors.ts#L86